pegasus/migrations/106_add_popularityrank_to_tutorials.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table(:tutorials) do
+      add_column :popularityrank, Integer, null: false
```

Comment:
I don't see that done for any of the other pegasus gsheet table migrations.  I think this is because we always fully ingest the gsheet, so this is more to protect against missing entries in the spreadsheet going forward.